datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/BaseE2ETest.java

```diff
@@ -51,6 +52,7 @@
 @Slf4j
 @RequiredArgsConstructor
 public abstract class BaseE2ETest extends WithStorageProvider {
+    protected Charset THE_CHARSET=Charset.forName("UTF-8");
```

Comment:
Instead of THE_CHARSET it is possible to import statically UTF_8 from StandardCharsets